static/js/search_suggestion.js

```diff
@@ -109,10 +110,22 @@ function get_stream_suggestions(last, operators) {
     const hilite = typeahead_helper.highlight_with_escaping_and_regex;
 
     const objs = streams.map((stream) => {
-        const prefix = "stream";
         const highlighted_stream = hilite(regex, stream);
-        const verb = last.negated ? "exclude " : "";
-        const description = verb + prefix + " " + highlighted_stream;
+        let description = "";
+        // Here escape value is set to false so that stream names with some letters as highlighted
+        // are rendered correctly otherwise suggestions like Exclude stream verona are rendered as
+        // Exclude stream <strong>v</strong><strong>e</strong>...
```

Comment:
I think it would be nice to choose a more appropriate name for `highlighted_stream` so that we don't need to explain in a separate comment why we should not escape it. Something like `highlighted_stream_name_in_html` or something appropriate.

Reply:
Made these changes.